addons/web/static/src/js/views/kanban/search_panel.js

```diff
@@ -30,6 +30,8 @@ var SearchPanel = Widget.extend({
      * @param {Object} params.fields
      * @param {string} params.model
      * @param {Object} params.sections
+     * @param {boolean} params.preselectFirstRecord if true, the category tree will default on
```

Comment:
This looks more like an option than data.
I think it would be best to change signature of `init` like this:
```js
/**
 * ...
 * @param {Object} [options={}]
 * @param {boolean} [options.preselectFirst=false] if true, the category
 *   tree will default on the 1st record instead of "All"
 */
init: function (parent, data, options) {
   // ...
   this.options = options || {};
},
```

Where `data` is simply a rename for `params`.